web-animations-2/Overview.bs

```diff
@@ -6,6 +6,14 @@
   color: #005A9C;
 }
 
+dl.switch  > [data-md] > p {
+  display: inline;
+}
+
+dl.switch > dt > ul > li {
+  text-indent: 0;
+}
+
```

Comment:
@tabatkins there seems to be issues with the generated bikeshed css when we switched to dl instead of just the div. You can see the issue at this [link](https://drafts.csswg.org/web-animations-2/#setting-the-current-time-of-an-animation#:~:text=Update%20either%20animation%E2%80%99s%20hold%20time%20or%20start%20time%20as%20follows) which @kevers-google is trying to work around here. Do you know if there's something wrong with the spec formatting or generator?

Reply:
Could you provide more details on what's wrong? I see that the "switch icon" is on its own line (unavoidable given the way it's injected, since the dt contains block elements), but it doesn't look like that's what your CSS is trying to mess with.

Reply:
The bullets in the list are also incorrectly indented. See https://drafts.csswg.org/web-animations-1/#setting-the-current-time-of-an-animation for comparison. web-animations-1 seems to be using <div class="switch">: https://github.com/w3c/csswg-drafts/blob/main/web-animations-1/Overview.bs#L957

Reply:
The switch being on it's own line was resolved by the rule:

dl.switch  > [data-md] > p {
  display: inline;
}

This aligns with a rule we have in web-animations-1, namely:

div.switch > dl > dt > p {
  display: inline;
}

The overlap resulted from the following rules in base.css:

dl.switch > dt {
    text-indent: -1.5em;
}

When applied to a paragraph inside a list item, it creates an overlap between the paragraph and the marker.

web-animation-1 controls the indent within a switch via these rules:

div.switch > dl > dt:before {
  margin-left: -1.3em;
}
div.switch > dl > dt > p {
  display: inline;
}

Reply:
Oh, huh, I see that `base.css` is doing some weird text-indent stuff. I... don't understand why it's doing that; it appears to only work well on single-line `dt`s, and makes structured ones like this look weird. @fantasai, I don't remember if this style was imported from Bikeshed's stylesheet or done by you, any idea?

Reply:
Moving the style changes to a separate PR to unblock the remainder of the changes.

Reply:
Style changes moved to https://github.com/w3c/csswg-drafts/pull/6655.